mbld/main.myr

```diff
@@ -80,8 +82,9 @@ const main = {args : byte[:][:]
 		if cmd.args.len == 0
```

Comment:
Looks good, but have you tested it with no arguments? I think this will give an error -- you'll probably want srcs here?

Reply:
Do you mean the initialization of src?
I initialize it at https://github.com/oridb/mc/blob/97aaecc00f6e96c3b7a5ce7c323f0df70685572f/mbld/main.myr#L42

Reply:
I tested with `make` and run mbld in `mbld/` . I didn't find any error so far.

Reply:
I mean that if I run a command like `mbld -R foo.myr`, then `cmd.args` will be empty, because `foo.myr` got taken as an option for `-R`, instead of as an argument, and we will get the error `std.fatal("nothing to compile\n")`

Reply:
Fixed